tests/python/unittest/test_meta_schedule_task_scheduler.py

```diff
@@ -337,6 +338,23 @@ def test_meta_schedule_task_scheduler_override_next_task_id_only():  # pylint: d
         )
 
 
+def should_skip_oob_test():
+    flags = libinfo()
+    # TODO(gigiblender) This combination of flags should be enabled only in the ci_minimal CI configuration.
+    #  Remove this once the OOB test is fixed.
+    return (
+        flags["USE_RELAY_DEBUG"] == "ON"
+        and flags["USE_LLVM"] == "llvm-config"
+        and flags["USE_MICRO"] != "ON"
+    )
+
+
+@pytest.mark.skipif(
```

Comment:
Hi, can you please enable this test after rebase to latest main because it’s been fixed, thanks!